docs/user_manual/dataset-terminology.md

```diff
@@ -42,30 +47,30 @@ The examples in brackets are given in context of a dataset of a `line` component
 
 - **Scenario:** A single time step / switch state topology.
 
-- **Batch:** The total set of scenarios. (there is only one batch)
+- **Batch:** The set of scenarios. (there is only one batch)
 
 - **Batch size:** The total number of scenarios in the batch.
 
 - **n_scenarios:** The total number of scenarios in the batch. (Same as Batch Size)
 
 - **n_component_elements_per_scenario:** The number of elements of a specific component for each scenario. This can be an integer (for dense batches), or a list of integers for sparse batches, where each integer in the list represents the number of elements of a specific component for the scenario corresponding to the index of the integer.
 
-- **Sub-batch:** Only used internally, in the C++ code, when all scenarios in a batch calculation are distributed over multiple threads, so that each thread can handle a sub-batch, to utilize the calculation power of multi-core processors.
+- **Sub-batch:** When computing in parallel, all scenarios in batch calculation are distributed over threads. Each thread handles a subset of the `Batch`, called a `Sub-batch`. This is only used internally in the C++ code.
 
 ## Attributes of Components
 
-| Attribute characteristic | Description                                                                                                                                                                                                                                                                                                  |
+| Attribute |  Description  |
 | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | name                     | Name of the attribute. It is exactly the same as the attribute name in `power_grid_model.power_grid_meta_data`.                                                                                                                                                                                              |
-| data type                | Data type of the attribute. It is either a type from the table in [Native Data Interface](../advanced_documentation/native-data-interface.md#basic-data-types). Or it can be an enumeration as above defined. There are two special data types `RealValueInput` and `RealValueOutput` which are independent. |
+| data type                | Data type of the attribute. It is either a type from the table in [Native Data Interface](../advanced_documentation/native-data-interface.md#basic-data-types), or an enumeration as defined above. There are two special data types that are independent from one another; `RealValueInput` and `RealValueOutput`. |
```

Comment:
```suggestion
| data type                | Data type of the attribute. It is either a type from the table in [Native Data Interface](../advanced_documentation/native-data-interface.md#basic-data-types), or an enumeration as defined above. There are two special data types that are independent from one another, namely `RealValueInput` and `RealValueOutput`. |
```

Reply:
Included in the latest commit. Conversation resolved.